UPDATING.md

```diff
@@ -163,7 +163,7 @@ The following table shows changes in import paths.
 |airflow.contrib.hooks.datastore_hook.DatastoreHook                                                                |airflow.gcp.hooks.datastore.DatastoreHook                                                                  |
 |airflow.contrib.hooks.gcp_bigtable_hook.BigtableHook                                                              |airflow.gcp.hooks.bigtable.BigtableHook                                                                    |
 |airflow.contrib.hooks.gcp_cloud_build_hook.CloudBuildHook                                                         |airflow.gcp.hooks.cloud_build.CloudBuildHook                                                               |
-|airflow.contrib.hooks.gcp_compute_hook.GceHook                                                                    |airflow.gcp.hooks.compute.GceHook                                                                          |
+|airflow.contrib.hooks.gcp_compute_hook.GceHook                                                                    |airflow.gcp.hooks.compute.ComputeEngineOperator                                                                          |
```

Comment:
Shouldn't be `ComputeEngineHook ` instead of `ComputeEngineOperator`?
I am not sure is it allowed to change some older informations. Maybe it should be done in separate information. WDY @nuclearpinguin ?

Reply:
It is recommended to change the old information if it is no longer correct. It is not advisable to make changes to UPDATING.MD in versions that have already been released, unless it corrects an error in the description. This note has been deliberately written in a manner that will allow the changes envisaged by AIP-21 to be made freely.

Reply:
I think it's quite justified to update "UPDATING.md" . It has not been released in 2.0 and we are not going to backport it to 1.10.* so it's ok to make it the "real" change here - even if we have to change it. We are going to change gcp to google soon and it's better to update UPDATING.md for that as well.

Reply:
So, should I change the path in UPDATING.md, here https://github.com/apache/airflow/blob/master/UPDATING.md?